extensions/compose/src/extension.ts

```diff
@@ -50,7 +89,44 @@ async function postActivate(extensionContext: extensionApi.ExtensionContext): Pr
 }
 
 export async function deactivate(): Promise<void> {
+  stopMonitoringLoop = true;
   if (composeExtension) {
     await composeExtension.deactivate();
   }
 }
+
+// Monitor the provider and check to see if the binary has been installed system-wide or not,
+// if it has, then we can update the status to "installed" and the user can use the "podman compose" command
+async function monitorProvider(
```

Comment:
Added this as well, which will monitor if it's installed system wide every 8 seconds and update the provider accordingly.